src/EPPlus/Properties/AssemblyInfo.cs

```diff
@@ -46,4 +46,8 @@
     //[assembly: AssemblyVersion("5.2.1.0")]
     //[assembly: AssemblyFileVersion("5.2.1.0")]
 #endif
-[assembly: AllowPartiallyTrustedCallers]
\ No newline at end of file
+[assembly: AllowPartiallyTrustedCallers]
+
+#if !NET35
+[assembly: SecurityRules(SecurityRuleSet.Level1)]
```

Comment:
Why this change?

Reply:
the security policy in the .Net framework did not allow some code to execute, I was getting this error Method `"OfficeOpenXml.Utils.RecyclableMemory.GetStream ()" could not access method "OfficeOpenXml.Utils.RecyclableMemory.get_MemoryManager ()".` at https://github.com/EPPlusSoftware/EPPlus/pull/207/files?file-filters%5B%5D=.cs#diff-88ed0abfe303a02040995b05494c6bf79414df00ecb89f678f8eab16d7d6f092R33
As far as i understood it is due to missing `[assembly: AllowPartiallyTrustedCallers]` in **Microsoft.IO.RecyclableMemoryStream** library

Reply:
Yes, I see, but this setting revert the security back to .NET 2.0 and should not be used. 
https://docs.microsoft.com/en-us/dotnet/framework/misc/security-transparent-code-level-1
I have opened an issue to see if this can be changed... https://github.com/microsoft/Microsoft.IO.RecyclableMemoryStream/issues/108

Reply:
Subscribed to notifications, I will follow the issue

Reply:
Immediately after the adoption of the [PR](https://github.com/microsoft/Microsoft.IO.RecyclableMemoryStream/pull/110), I will remove this lines